package.json

```diff
@@ -21,7 +21,7 @@
         "stylelint": "stylelint '**/*.less'",
         "lint": "eslint --cache --cache-location node_modules/.cache/eslint",
         "prettier": "prettier '**/*.{svg,yml,js,ts,html,md,less,json}' --write",
-        "typecheck": "tsc --noEmit --skipLibCheck --project tsconfig.spec.json",
```

Comment:
Why this change?

Reply:
because for `incremental` build these options don't work

Reply:
But this command is just to check for ts errors in precommit. This will increase commit time. Maybe we need a separate tsconfig for it?

Reply:
fixed